h/templates/client/annotation.html

```diff
@@ -174,23 +174,7 @@
         h-tooltip>
         <i class="h-icon-annotation-reply btn-icon "></i>
       </button>
-      <span class="share-dialog-wrapper" ng-if="!vm.feature('direct_linking')">
-        <button class="btn btn-clean annotation-action-btn"
-          ng-click="vm.showShareDialog = !vm.showShareDialog"
-          aria-label="Link"
-          h-tooltip>
-          <i class="h-icon-link btn-icon "></i>
-        </button>
-        <span class="share-dialog share-dialog--actions" ng-click="$event.stopPropagation()">
```

Comment:
The `share-dialog--actions` and `share-dialog__link` classes will be unused once this is removed, so they can be removed from `annotations.scss`